backend/store/deploy.go

```diff
@@ -242,3 +242,160 @@ func (*Store) upsertDeploymentConfigImpl(ctx context.Context, tx *Tx, upsert *ap
 	}
 	return &cfg, nil
 }
+
+// FindDeploymentConfigMessage is the message to find a deployment config.
+type FindDeploymentConfigMessage struct {
```

Comment:
Instead of introducing Find, let's just use (projectUID int) for GetDeploymentConfig?

Reply:
Done.